src/System.Private.ServiceModel/src/System/IdentityModel/Selectors/X509CertificateValidator.cs

```diff
@@ -67,8 +67,6 @@ public ChainTrustValidator()
 
             public ChainTrustValidator(bool useMachineContext, X509ChainPolicy chainPolicy, uint chainPolicyOID)
             {
-                Contract.Assert(useMachineContext == false, "CoreCLR does not have ctor allowing useMachineContext = true");
```

Comment:
Reviewers: @iamjasonp confirmed this Assert was no longer needed, and in fact must be removed because it overconstrains and fires.